src/Admin/Menu/MenuNodeAdmin.php

```diff
@@ -69,14 +70,24 @@ protected function configureFormFields(FormMapper $formMapper)
                                 'content' => 'content',
                             ),
                             'map' => array(
-                                'route' => array('link'),
+                                'route' => array('link','routeParameters'),
                                 'uri' => array('link'),
                                 'content' => array('content', TreeSelectType::class),
                             ),
                             'placeholder' => 'auto',
                             'required' => false,
                         ))
                         ->add('link', TextType::class, array('required' => false, 'mapped' => false))
+                        ->add('routeParameters', KeyValueType::class, array(
```

Comment:
currently the key-value bundle is an optional dependency. even if we would make it a hard dependency, the user still needs to load the bundle in the kernel.

i think we should have an options array in the constructor to choose whether route parameters should be editable. then the DI extension can check whether the keyvaluebundle is loaded and complain if its not. the seo bundle form type does this, see [the form](https://github.com/symfony-cmf/seo-bundle/blob/5900c572b10f9df58689e898feb1970bd381ef43/src/Form/Type/SeoMetadataType.php#L68) and [the DI extension](https://github.com/symfony-cmf/seo-bundle/blob/5900c572b10f9df58689e898feb1970bd381ef43/src/DependencyInjection/CmfSeoExtension.php#L390-L397) and [configuration](https://github.com/symfony-cmf/seo-bundle/blob/5900c572b10f9df58689e898feb1970bd381ef43/src/DependencyInjection/Configuration.php#L298-L306)

can you please add the configuration for this?